client/my-sites/patterns/components/pattern-library/style.scss

```
@@ -12,6 +12,10 @@
 	top: 0;
 	z-index: 3;
 
+	&.pattern-library__pill-navigation--sticky {
+		box-shadow: 0 3px 8px 0 rgba(0, 0, 0, 0.12), 0 3px 1px 0 rgba(0, 0, 0, 0.04);
```

```suggestion
		box-shadow: 0 0 8px hsla(0, 0%, 0%, 0.08);
```
I think the box shadow looks noticeably more dramatic than what we have for `.pattern-preview__renderer` right now. Do you think we could change it to something like this?

---

Let's go with it, changed